synapse/lib/cell.py

```diff
@@ -1329,36 +1352,20 @@ async def iterBackupArchive(self, name, user):
 
         await self.boss.promote('backup:stream', user=user, info={'name': name})
 
-        ctx = multiprocessing.get_context('spawn')
-        done = ctx.Queue()
-
-        proc = None
         mesg = 'Streaming complete'
 
-        def getproc():
-            proc = ctx.Process(target=_iterBackupProc, args=(path, linkinfo, done))
-            proc.start()
-            return proc
-
         try:
-            proc = await s_coro.executor(getproc)
+            fut = self.procTask(_iterBackupProc, path, linkinfo)
```

Comment:
this is a POC port to using the procpool as opposed to making a new process on demand